### PR TITLE
Rename to bringit

### LINKED ISSUE #30: Rename gem to Bringit and publish to rubygems

### PR DESCRIPTION
**Pull request by phyrog**
Closes #30. Not ready for review yet. Test are passing however.

What shall I do with files that concern `gitlab_git` (Changelog, git data)?

### CODE REVIEW

**Comment by phyrog** [comment]
This should be ready for review now. I reset the version to `1.0.0` and removed the changelog to avoid confusion. There are some occurences of `gitlab` left, but these must remain (unless I missed something importent), as they reference some repositories needed for the tests.

File: bringit.gemspec [hunk 1]
```diff
@@ -3,20 +3,20 @@
...
-  s.name        = 'gitlab_git'
-  s.version     = Gitlab::Git::VERSION
+  s.name        = 'bringit'
+  s.version     = Bringit::VERSION
   s.date        = Time.now.strftime("%Y-%m-%d")
-  s.summary     = "Gitlab::Git library"
-  s.description = "GitLab wrapper around git objects"
+  s.summary     = "Bringit library"
+  s.description = "Bringit wrapper around git objects"
   s.authors     = ['Dmitriy Zaporozhets']
```
Comment (eugenk) [inline bringit.gemspec]: I think we should add us as the authors.

File: bringit.gemspec [hunk 2]
```diff
@@ -3,20 +3,20 @@
...
   s.email       = 'dmitriy.zaporozhets@gmail.com'
```
Comment (eugenk) [inline bringit.gemspec]: ... and change the email address.

File: spec/support/seed_repo.rb [hunk 3]
```diff
@@ -9,7 +9,7 @@
 # fa1b1e6c004a68b7d8763b86455da9e6b23e36d6 Merge branch 'add-files' into 'master'
 # eb49186cfa5c4338011f5f590fac11bd66c5c631 Add submodules nested deeper than the root
 # 18d9c205d0d22fdf62bc2f899443b83aafbf941f Add executables and links files
-# 5937ac0a7beb003549fc5fd26fc247adbce4a52e Add submodule from gitlab.com
```
Comment (eugenk) [inline spec/support/seed_repo.rb]: This change seems odd.

Comment (phyrog) [reply spec/support/seed_repo.rb]: Oh, we didn't register `bringit.com` yet? :smile:

File: bringit.gemspec [hunk 4]
```diff
@@ -11,8 +11,8 @@ Gem::Specification.new do |s|
   s.date        = Time.now.strftime("%Y-%m-%d")
   s.summary     = "Bringit library"
   s.description = "Bringit wrapper around git objects"
-  s.authors     = ['Dmitriy Zaporozhets']
-  s.email       = 'dmitriy.zaporozhets@gmail.com'
+  s.authors     = ['Dmitriy Zaporozhets', 'Ontohub Core Developers']
+  s.email       = ['dmitriy.zaporozhets@gmail.com', 'ontohub-dev-l@ovgu.de']
```
Comment (eugenk) [inline bringit.gemspec]: Please reverse the order.

Comment (phyrog) [reply bringit.gemspec]: Done in 3a14df6

**Pull request merged**